book/example_6_surface.md

```diff
@@ -0,0 +1,163 @@
+# Graph Un-Reachability
```

Comment:
Will take a look at this & rest of the files in a separate review

Reply:
I'm thinking that this example isn't helpful until we can do it with a module system. It's basically a reuse of the reachability spec, followed by a `difference` op. This would be pretty with modular reuse, but for now doesn't add much (especially since we already discussed stratification).

Reply:
I think it is still helpful even if it is not _super_ helpful